### ISSUE TITLE
L32 with CAM6 physics

### ISSUE THREAD

**Issue by adamrher** [NCAR/amwg_dev#531]
To test if precip biases in E.Pac are due to cam_dev physics.

___

Identical to the recent L32 run (https://github.com/NCAR/amwg_dev/issues/524), but substitute %CAM60 for %CAM_DEV in a long name compset that is otherwise identical to BLT1850. Does it this way (rather than just setting -phys cam6) will ensure that the gust mods aren't turned on. 

___
**Case directory:**
- Locally (if still available):
`/glade/campaign/cesm/cesmdata/cseg/runs/cesm2_0/$CASE`

- On github:
https://github.com/NCAR/amwg_dev/tree/$CASE
___
**Sandbox:**
- Locally (if still available): `/glade/work/hannay/cesm_tags/cesm2_3_alpha16g`
 
- On github: https://github.com/ESCOMP/cesm/tree/cesm2_3_alpha16g
___
**Diagnostics:**
- Diags (if available):
[https://webext.cgd.ucar.edu/BLT1850/$CASE/](https://webext.cgd.ucar.edu/BLT1850/$CASE/)
___
**Output:**
- Raw output on derecho (if still available): `/glade/derecho/scratch/hannay/archive/$CASE/`
- CAM Climos: `/glade/campaign/cgd/amp/amwg/climo/$CASE`
___
**Contacts:**
@cecilehannay
___
**Extra details:**

**Comment by cecilehannay** [comment]
Hi Adam,
The longname in (https://github.com/NCAR/amwg_dev/issues/524) uses `CAM%DEV%LT%GHGMAM4`
Do you want to replace this by `CAM60%LT%GHGMAM4` or just `CAM60.`

**Comment by cecilehannay** [comment]
Also, there was some namelist mods in (https://github.com/NCAR/amwg_dev/issues/524). Do we want these or not? 
```
! dust tuning
dust_emis_fact = 1.3

! clubb diffusion on
clubb_l_do_expldiff_rtm_thlm = .true.

! tuning
microp_aero_wsub_scale  = 1.0
clubb_C8=4.25

! revert NorESMs mods
cldfrc2m_do_avg_aist_algs = .false.
cldfrc2m_qist_min = 1.e-7

bnd_topo = "/glade/campaign/cgd/amp/pel/topo/files/se/ne30pg3_gmted2010_modis_bedmachine_nc3000_Laplace0100_noleak_20240117.nc"
se_hypervis_subcycle = 2
```

**Comment by adamrher** [comment]
Yes, I would keep those settings except I would not explicitly set ```dust_emis_fact``` or ```clubb_C8```. Let's just run with the default values for those two parameters. Thanks!

**Comment by cecilehannay** [comment]
Sounds good. Also, I have skip all the SourceMods/src.cam from (https://github.com/NCAR/amwg_dev/issues/524).
They were related to gustiness, pumas and fast dycore. 

Last thing, here is the way I set the longname.
I replaced:
`CAM%DEV%LT%GHGMAM4` -> `CAM60`
 
I want to make sure you didn't want: `CAM60%LT%GHGMAM4`

**Comment by adamrher** [comment]
Correct, just ```CAM60```. None of that other stuff.
